cmd/clusterctl/internal/util/resource_tuples_test.go

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"testing"
+
+	. "github.com/onsi/gomega"
+)
+
+func TestResourceTypeAndNameArgs(t *testing.T) {
+
+	tests := []struct {
+		name    string
+		args    []string
+		want    []ResourceTuple
+		wantErr bool
+	}{
+		{
+			name: "valid",
+			args: []string{"machinedeployment/foo"},
```

Comment:
What happens if we just specify `machinedeployment`? What is the expected behavior?
That is, if the user does `clusterctl alpha rollout restart machinedeployment`.

Reply:
Added a new test case.

Reply:
So I meant what if we just specified `machinedeployment` without the slash.

If we remove the `/` on line 61 below, we don't get an error but do we expect any `ResourceTuple`s?

That is, if we run `clusterctl alpha rollout restart machinedeployment` are we expecting all the machinedeployments in the specified namespace to rollout or do we error asking the user to specify a name for the machinedeployment?

Reply:
I see your point. Yes, we should throw an error to the user.